### ISSUE TITLE
Would like to have control over padding and text alignment within the dropdown control please.

### ISSUE THREAD

**Issue by busidex-vinbrown** [enisn/Xamarin.Forms.InputKit#72]
The dropdown control has a lot of wasted space. I would like to be able to control the padding, text alignment and text size. Controlling the text size via the Global Settings does not work as indicated in the readme.txt file.

**Comment by enisn** [comment]
Do you need that control smaller ?

**Comment by busidex-vinbrown** [comment]
Yes, it's much too big for my UI. I'm using it in a list of phone numbers. The dropdown list (on each row) is for selecting a phone number type. So the row consists of the dropdownlist, the phone number, and a delete icon.

**Comment by busidex-vinbrown** [comment]
I actually have a version of it working locally that works for me. I pulled down the repo, and was going to submit a PR, but don't have permissions.

**Comment by enisn** [comment]
@busidex-vinbrown 
You can create Pull request like [others](https://github.com/enisn/Xamarin.Forms.InputKit/pulls?q=is%3Apr+is%3Aclosed).  I didn't block any Pull Request
